plugins/modules/dhcp_ha_group.py

```diff
@@ -135,6 +113,18 @@
           - host: "{{ host_2.id }}"
             role: "active"
         state: present
+        
+    - name: Create DHCP HA Group with mode anycast
+      infoblox.universal_ddi.dhcp_ha_group:
+        name: "example_ha_group"
+        mode: "anycast"
+        anycast_config_id: "{{ ac_config.id }}"
+        hosts:
+          - host: "{{ host_1.id }}"
```

Comment:
```suggestion
          - host: "{{ host_info_1.objects[0].id }}"
```
Please fix this for all examples .

Reply:
done